device_doctor/lib/src/health.dart

```diff
@@ -26,3 +78,25 @@ class HealthCheckResult {
     return '$buf';
   }
 }
+
+/// Check healthiness for discovered devices.
+///
+/// If any failed check, an explanation message will be sent to stdout and
+/// an exception will be thrown.
+Future<void> healthcheck(Map<String, List<HealthCheckResult>> deviceChecks) async {
+  if (deviceChecks.isEmpty) {
+    stderr.writeln('No healthy device is available');
+    throw StateError('No healthy device is available');
```

Comment:
The bot config should already write the error when it is thrown, why are we also logging it?

Reply:
Added for testing. Removed.